### PR TITLE
Handle image upload errors on user creation and update

### LINKED ISSUE #59: Users: Handle image upload errors and delete associated User

### PR DESCRIPTION
**Pull request by averyfairbanks**
Deletes new user if profile picture cannot be uploaded. Users can be updated but if there is an error uploading an image it will remain in the previous state.

Closes #59

### CODE REVIEW

**Review by forrestwilkins — APPROVED** [review]

File: src/apollo/resolvers/user.ts [hunk 1]
```diff
@@ -11,7 +11,14 @@ import Messages from "../../utils/messages";
 
 const saveProfilePicture = async (user: any, image: any) => {
   if (image) {
-    const path = await saveImage(image);
+    let path;
+
+    try {
+      path = await saveImage(image);
+    } catch (e) {
+      throw new ApolloError(" Unable to upload image(s)\nError response: " + e);
```
Comment (forrestwilkins) [inline src/apollo/resolvers/user.ts]: Can we move this to where the `return e;` statement currently is in `signUp`? I'm thinking we can get the same error output by only having the try catch and throw in `signUp`.

File: src/apollo/resolvers/user.ts [hunk 2]
```diff
@@ -11,7 +11,8 @@ import Messages from "../../utils/messages";
 
 const saveProfilePicture = async (user: any, image: any) => {
   if (image) {
-    const path = await saveImage(image);
+    let path = await saveImage(image);
```
Comment (forrestwilkins) [inline src/apollo/resolvers/user.ts]: Can we change this back to const?

Comment (averyfairbanks) [reply src/apollo/resolvers/user.ts]: Handled in most recent commit 6d20978

File: src/locales/en.ts [hunk 3]
```diff
@@ -28,6 +28,7 @@ const en = {
 
   errors: {
     somethingWrong: () => "Something went wrong...",
+    imageUploadError: () => " Unable to upload image(s)",
```
Comment (forrestwilkins) [inline src/locales/en.ts]: Can we remove the space before Unable?

**Pull request merged**